erdos/op.py

```diff
@@ -50,6 +46,9 @@ def execute(self):
         """
         pass
 
+    def send(self, stream_name, msg):
```

Comment:
Can you please do now make this change in this PR? This change is unrelated to the bug, and makes the PR much larger. Plus, the examples are not going to work as a result of this change. I think it's better to separate this change into a new PR, in which we also update the examples. However, I would put this new PR as low priority for now.